app/assets/stylesheets/custom.css.scss

```diff
@@ -141,28 +135,28 @@ h1 {
   background-color: darkgrey;
 }
 
-// table td {
-//   font-family: monospace;
-//   border: 2px solid black;
-//   padding: 10px;
-//   align: center;
-// }
-//
-// .admin_orders_list {
-//   font-family: monospace;
-//   width: 65%;
-//   padding: 5px;
-//   border: 1px solid;
-//   height: 300px;
-//   overflow: auto;
-// }
-//
+table td {
+  font-family: monospace;
+  border: 2px solid black;
+  padding: 10px;
+  align: center;
+}
+
+.admin_orders_list {
+  font-family: monospace;
```

Comment:
Properties should be ordered border, font-family, height, overflow, padding, width